lib/ukrandom/README.md

```diff
@@ -0,0 +1,44 @@
+# libukrandom: Cryptographically Secure Randomness
+
+This library implements a cryptographically-secure PRNG (CSPRNG).
+Secure randomness is available to applications via the `get_random()` syscall (`CONFIG_LIBUKRANDOM_GETRANDOM`), and via `/dev/random`, `/dev/urandom`, and `/dev/hwrng` (`CONFIG_LIBUKRANDOM_DEVFS`).
```

Comment:
```suggestion
Secure randomness is available to applications via the `getrandom()` syscall (`CONFIG_LIBUKRANDOM_GETRANDOM`), and via `/dev/random`, `/dev/urandom`, and `/dev/hwrng` (`CONFIG_LIBUKRANDOM_DEVFS`).
```

Isn't the system call [`getrandom()`](https://man7.org/linux/man-pages/man2/getrandom.2.html), with the underline?

Reply:
Good catch, thanks for spotting it out. I pushed a fix.